src/main.rs

```diff
@@ -42,27 +19,41 @@ impl Op {
 }
 
 #[derive(Clone, Debug, Bpaf)]
+enum Op {
+    #[bpaf(command)]
+    /// Get something
+    Get {
+        /// help here
+        #[bpaf(positional)]
```

Comment:
it is really good to see how effectively bpaf can be used. i was parsing the parameters programmatically and imperatively.